modules/solace/go.mod

```
@@ -0,0 +1,68 @@
+module github.com/testcontainers/testcontainers-go/modules/solace
+
+go 1.23.0
+
+require (
+	github.com/docker/docker v28.2.2+incompatible
+	github.com/docker/go-connections v0.5.0
+	github.com/stretchr/testify v1.10.0
+	github.com/testcontainers/testcontainers-go v0.37.0
```

Suggestion: we released a new version yesterday. Please run mod tidy after that.
```suggestion
	github.com/testcontainers/testcontainers-go v0.38.0
```

---

done